multiversx_sdk_core/transaction_factories/smart_contract_factory_test.py

```diff
@@ -0,0 +1,88 @@
+from pathlib import Path
+
+from multiversx_sdk_core.address import Address, compute_contract_address
+from multiversx_sdk_core.constants import CONTRACT_DEPLOY_ADDRESS, DEFAULT_HRP
+from multiversx_sdk_core.transaction_factories.smart_contract_factory import \
+    SmartContractFactory
+from multiversx_sdk_core.transaction_factories.transaction_factory_config import \
+    TransactionFactoryConfig
+
+
+class TestSmartContract:
+    config = TransactionFactoryConfig("D")
+    factory = SmartContractFactory(config)
+
+    def test_compute_contract_address(self):
```

Comment:
Test can be removed (there should be an existing one in `address_test.py` - if not, can be moved there).

Reply:
There is indeed a test in `address_test.py`. Removed the one I wrote.